application/views/admin/expressions/test/survey_logic_file.php

```diff
@@ -32,4 +32,27 @@
             </div>
         </div>
     </div>
-</div>
\ No newline at end of file
+</div>
+
+<style>
+    /**
+     * This whole style block is a workaround for https://bugs.limesurvey.org/view.php?id=18250
+     */
+    .main-content-container {
+        overflow-x: visible;
+    }
+    .table-responsive {
+        overflow-x: visible;
+    }
+
+    #vue-sidebar-container {
+        display: none;
+    }
+    #pjax-content {
+        max-width: 100%!important;
+    }
+    #vue-apps-main-container > .col-11 {
```

Comment:
No way to set as col-12 for logic file ?

Reply:
Not sure.

Reply:
I give a quick test and send the screenshot ;)

Reply:
Oh : like before : adding fix for CSS inline : less update.
Good idea :)